lib/Pill/index.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Icon from '../Icon';
+
+const Pill = props => {
+  return (
+    <span className={`pill ${props.className}`}>
+      {props.children}
+      {props.onClearClick ? (
```

Comment:
Couldn't this be composed via `props.children`? Imagine if we want to use this space for another type of action in the future. What do you think?

Reply:
This could then use any icon or any re-usable pattern within the library rather than a strictly defined one.

Reply:
Completely agree.